Mathlib/RingTheory/Localization/Defs.lean

```diff
@@ -528,6 +528,14 @@ theorem lift_injective_iff :
     Injective (lift hg : S → P) ↔ ∀ x y, algebraMap R S x = algebraMap R S y ↔ g x = g y :=
   (toLocalizationMap M S).lift_injective_iff hg
 
+variable (M) in
+include M in
```

Comment:
Is this how it works?

Reply:
Sure, let's merge it then.

Reply:
It seems there is no shorter way, see https://leanprover.zulipchat.com/#narrow/channel/113488-general/topic/include.20in.20with.20variable.20in.